ansible/configs/ansible-automation-platform/roles/vscode-server_cfg/tasks/nginx-server.yml

```diff
@@ -0,0 +1,81 @@
+---
+
+- name: Install nginx on bastion/control node RHEL 7
+  yum:
+    name:
+      - nginx
+      - certbot-nginx
+    state: present
+  when: ansible_distribution == "RedHat" and ansible_distribution_major_version == '7'
+
+- name: Install nginx on bastion/control node RHEL 8
+  block:
+    # would be better to use GPTE Satellite
+    - name: Enable EPEL
+      dnf:
+        name: "https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm"
+        state: present
+        disable_gpg_check: true
+
+    - name: install nginx/certbot
+      yum:
+        name:
+          - nginx
+          - certbot
+          - python3-certbot-nginx
+        state: present
+  when: ansible_distribution == "RedHat" and ansible_distribution_major_version == '8'
```

Comment:
if you fix the GPG check issue beforehand, you should be able to just use the `package` ansible module and ansible will pick the default package manager, right?